website/components/_app.tsx

```diff
@@ -58,7 +58,13 @@ const Analytics = () => (
       j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
       'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
     })(window,document,'script','dataLayer','GTM-MNQGLJBB');`}</Script>
-
+    <Script
+      type="text/javascript"
+      id="hs-script-loader"
+      async
+      defer
+      src="//js-na1.hs-scripts.com/43689758.js"
```

Comment:
Tried to use our own domain but got hit with https://community.cloudflare.com/t/error-1014-cname-cross-user-banned-how-to-resolve/324935